bin/release.sh

```diff
@@ -28,7 +28,8 @@ export AMO_SECRET=$(curl ${secret_url} | jq ".secret.AMO_SECRET" -r)
 # and also require it to be greater.
 # /!\ this changes install.rdf without commiting it.
 VERSION_SUFFIX=$(date +%Y%m%d%H%M)
-sed -i -E 's/em:version=\"([0-9.ab-]+)\"/em:version=\"\1.'$VERSION_SUFFIX'\"/g' install.rdf
+echo "VERSION_SUFFIX=$VERSION_SUFFIX"
```

Comment:
Was this logged for debugging? Maybe add a small echo before to say we are starting the release script or something?